### ISSUE TITLE
Code for Services Markdown Card has Error

### ISSUE THREAD

**Issue by pickerin** [dummylabs/thewatchman#93]
The code for the Services Markdown Card in the README.md incorrectly lists "services" as the attribute being pulled rather than "entities".  The correct code should look like this:
```
type: markdown
content: >-
  <h2 class="some"> <ha-icon icon='mdi:shield-half-full'></ha-icon> Watchman report</h2>
  <h3> Missing Services: {{ states.sensor.watchman_missing_services.state }} </h3>
  {%- for item in state_attr("sensor.watchman_missing_services", "entities") %}
  <hr><table><tr> <td>  <ha-icon icon='mdi:cloud-alert'></ha-icon> {{ item.id }}
  <a title="{{item.occurrences}}">{{item.occurrences.split('/')[-1].split(':')[0]}}</a>
  </td></tr></table>
  {%- endfor %}
card_mod:
  style:
    ha-markdown:
      $: |
        ha-markdown-element:first-of-type hr{
          border-color: #303030;
        }
```

**Comment by Nealtron** [comment]
@pickerin, can you confirm that your change is the same as the one in PR https://github.com/dummylabs/thewatchman/pull/69?

**Comment by pickerin** [comment]
This is the same issue as PR #69.

**Comment by Nealtron** [comment]
Thank you @pickerin, I've bumped the 1-year old merge request to fix the error.

**Comment by dummylabs** [comment]
Should be fixed together with #112 and #93 in 0.6.2
